src/contractTest/java/uk/gov/hmcts/reform/refdata/RefDataProfessionalOrganisationalInternalConsumerTest.java

```diff
@@ -0,0 +1,139 @@
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import au.com.dius.pact.consumer.dsl.DslPart;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import org.apache.http.client.fluent.Executor;
+import org.junit.After;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.OrganisationByProfileIdsRequest;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.OrganisationByProfileIdsResponse;
+import uk.gov.hmcts.reform.orgrolemapping.feignclients.PRDFeignClient;
+
+import java.util.List;
+
+import static io.pactfoundation.consumer.dsl.LambdaDsl.newJsonBody;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(SpringExtension.class)
+@ExtendWith(PactConsumerTestExt.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@PactFolder("pacts")
+@PactTestFor(providerName = "referenceData_organisationalInternal", port = "8090")
+@ContextConfiguration(classes = {RefDataCaseworkerConsumerApplication.class})
+@TestPropertySource(properties = {"feign.client.config.prdClient.url=http://localhost:8090"})
+@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class})
+public class RefDataProfessionalOrganisationalInternalConsumerTest {
+
+    private static final String PRD_GET_ORG_BY_PROFILE_URL
+            = "/refdata/internal/v1/organisations/getOrganisationsByProfile";
+    private static final String SEARCH_AFTER = "03b98806-8473-42c6-a6e3-004364a58b44";
+    private static final Integer PAGE_SIZE = 1;
+
+    @Autowired
+    PRDFeignClient prdFeignClient;
+
+    @BeforeEach
+    public void setUpEachTest() throws InterruptedException {
+        Thread.sleep(2000);
+    }
+
+    @After
+    void teardown() {
+        Executor.closeIdleConnections();
+    }
+
+    @Pact(provider = "referenceData_organisationalInternal", consumer = "accessMgmt_orgRoleMapping")
+    public RequestResponsePact getOrganisationsByProfileIdsWithPageSize(PactDslWithProvider builder)
+            throws JsonProcessingException {
+        return builder
+                .given("A page size & list of organisation profiles for a PRD internal organisation request")
+                .uponReceiving("A request for organisations")
+                .path(PRD_GET_ORG_BY_PROFILE_URL)
+                .query("&pageSize=" + PAGE_SIZE)
+                .body(new ObjectMapper().writeValueAsString(
+                        OrganisationByProfileIdsRequest.builder()
+                                .organisationProfileIds(List.of("SOLICITOR_PROFILE"))
+                                .build()
+                        ))
+                .method("POST")
+                .willRespondWith()
+                .status(200)
+                .body(buildOrganisationResponsePactDsl())
+                .toPact();
+    }
+
+    @Test
```

Comment:
I think we've decided to disable these pats for PR runs for now

Reply:
disabled in commit [9375d38](https://github.com/hmcts/am-org-role-mapping-service/pull/1743/commits/9375d389ba6ce2749085574411c4ea75aa7fd2f6)